tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -117,7 +135,7 @@ def rollback_parser_errors(datafile):
     num_deleted, models = ParserError.objects.filter(file=datafile).delete()
     logger.debug(f"Deleted {num_deleted} {ParserError}.")
 
-def parse_datafile_lines(datafile, program_type, section, is_encrypted):
+def parse_datafile_lines(datafile, program_type, section, is_encrypted, is_tribal):
```

Comment:
Why do we pass in `is_tribal` if it appears to be unused in this function? Also, isn't it a little redundant if we can determine that later from the datafile obj?